src/h5web/visualizations/heatmap/ColorBar.module.css

```diff
@@ -0,0 +1,26 @@
+.colorBar {
+  display: flex;
+  padding: 1em;
+}
+
+.colorBar[data-horizontal='false'] {
+  width: 6em;
+  min-height: 0;
+}
+
+.colorBar[data-horizontal='true'] {
```

Comment:
You can use a ternary with `undefined` to avoid having to specify `true`/`false` here (and instead use `[data-horizontal]` and `:not([data-horizontal])`. I find it a little nicer than to check a boolean string.

Reply:
Nice, I wanted to do so but didn't know the `:not` syntax :+1: